google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1/JsonToProtoMessageTest.java

```diff
@@ -598,7 +600,7 @@ public void testInt32NotMatchInt64() throws Exception {
           JsonToProtoMessage.INSTANCE.convertToProtoMessage(TestInt32.getDescriptor(), json);
       Assert.fail("should fail");
     } catch (IllegalArgumentException e) {
-      assertEquals("JSONObject does not have a int32 field at root.int.", e.getMessage());
+      assertTrue(e.getMessage().contains("JSONObject does not have a int32 field at root.int."));
```

Comment:
Why are these changes?

Reply:
Because we are directly throwing out index to row error message from convertToProtoMessage